posthog/tasks/exports/video_exporter.py

```
@@ -0,0 +1,224 @@
+            )
+            page = context.new_page()
+            record_started = time.monotonic()
+            try:
+                page.goto(url_to_render, wait_until="load", timeout=30000)
+            except PlaywrightTimeoutError:
+                pass
+            try:
+                page.wait_for_selector(wait_for_css_selector, state="visible", timeout=20000)
+            except PlaywrightTimeoutError:
+                pass
+            try:
+                page.wait_for_selector(".Spinner", state="detached", timeout=20000)
+            except PlaywrightTimeoutError:
+                pass
+            measured_width: Optional[int] = None
+            try:
+                dimensions = page.evaluate("""
+                    () => {
+                        const replayer = document.querySelector('.replayer-wrapper');
+                        if (replayer) {
+                            const rect = replayer.getBoundingClientRect();
+                            return {
+                                height: Math.max(rect.height, document.body.scrollHeight),
+                                width: replayer.offsetWidth || 0
+                            };
+                        }
+                        // Fallback for tables if no replayer
+                        const table = document.querySelector('table');
+                        return {
+                            height: document.body.scrollHeight,
+                            width: table ? Math.floor((table.offsetWidth || 0) * 1.5) : 0
+                        };
+                    }
+                """)
+                final_height = dimensions["height"]
+                width_candidate = dimensions["width"] or width
+                measured_width = max(width, min(1800, int(width_candidate)))
+                page.set_viewport_size({"width": measured_width, "height": int(final_height) + HEIGHT_OFFSET})
+            except Exception:
+                pass
```

Do we really mean to silently fail in case _any_ exception happens here?

---

resizing in replay is quite normal thing, but yep, will add a log